README.md

```diff
@@ -115,73 +123,77 @@ It creates:
 - A Load Balancer
 - Related IAM [Roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) and [Policies](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html).
 
+</details>
 
-<a name="fargate"></a>
-### Fargate
+### AWS Fargate
 
 [AWS Fargate](https://aws.amazon.com/fargate/) is part of [Amazon Elastic Container Service (ECS)](https://aws.amazon.com/ecs/). It's a managed service for running container-based applications, without having to worry about the underlying servers--sort of like [Lambda](https://aws.amazon.com/lambda/) for containers.
 
-The **_fargate.cfn.yml_** template creates:
+Creating a Fargate stack requires you to have first created a [VPC](#vpc) stack, and to enter the name of the VPC stack as the NetworkStackName parameter.
+
+<details>
+	<summary>Resources Created</summary>
 
 - An S3 bucket for the container
```

Comment:
This S3 bucket is a sample bucket used to showcase roles in containers/Fargate. The container is stored in ECR.